tensorflow/compiler/xla/service/gpu/done_event_thunk.cc

```diff
@@ -0,0 +1,58 @@
+#include "tensorflow/compiler/xla/service/gpu/done_event_thunk.h"
+
+namespace xla {
+namespace gpu {
+
+using SpmdDoneEvents = absl::flat_hash_map<int, std::unique_ptr<se::Event>>;
+absl::flat_hash_map<int, SpmdDoneEvents*> all_done_events;
+
+void ResetEvents(size_t num_output) { all_done_events.clear(); }
+
+StatusOr<std::vector<std::unique_ptr<se::Event>>> TakeDoneEvents(
+    size_t output_index, int devices) {
```

Comment:
`devices` -> `num_devices`?

Reply:
will do if you do not have other concerns. or feel free to do any update if you want